ci/ansible/roles/pulp-smash/defaults/main.yml

```diff
@@ -9,3 +9,9 @@ pulp_smash_password: "admin"
 
 # If SSL certificate should be verified
 pulp_smash_verify: "false"
+
+# Pulp Server version
+# pulp_smash_version variable is optional, if you want to specify it then you
+# can do so by passing additional variables to ansible-playbook command or by
+# uncommenting the following line and editing as needed.
+# pulp_smash_version: 2.7
```

Comment:
The interactive prompt is probably out of the question since I assume we want to automate this, but here a link for reference:

https://docs.ansible.com/ansible/playbooks_prompts.html